lib/utils/parseRequest.js

```diff
@@ -1,3 +1,13 @@
 module.exports = rawRequest => {
```

Comment:
The request body starts directly after a blank line. With your solution, you are expecting the request body to start on line 4, which won't always be the case

```js
module.exports = rawRequest => {
  const lines = rawRequest.split('\n');
  const [method, path] = lines[0].split(' ');

  const body = rawRequest.split('\r\n\r\n')[1];

  return {
    method,
    path,
    body
  };
};
```